index.html

```diff
@@ -1411,20 +1411,23 @@ <h3>Context Injection</h3>
           </p>
 
           <p>
-When securing a document, if an `@context` property is not provided in the document
-or the Data Integrity terms used in the document are not mapped by
-existing values in the `@context` property, implementations MUST inject or add
-an `@context` property with a value of
-`https://w3id.org/security/data-integrity/v2`.
+When an application is securing a document, if an `@context` property is not
+provided in the document or the Data Integrity terms used in the document are
```

Comment:
```
When an application is securing a document, if an `@context` property is not
provided in the document or the Data Integrity terms used in the document are
```

Excuse me if I am overly picky at this point...

Per the [Interpreting JSON as JSON-LD section](https://www.w3.org/TR/json-ld11/#interpreting-json-as-json-ld) of the JSON-LD spec, it is possible to transfer the context through an HTTP Link Header. This means that the term "provided in the document" may not be absolutely precise.

Interestingly, the aforementioned JSON-LD section also includes this statement:

> Please note that [JSON-LD documents](https://www.w3.org/TR/json-ld11/#dfn-json-ld-document) served with the application/ld+json media type MUST have all context information, including references to external contexts, within the body of the document. Contexts linked via a http://www.w3.org/ns/json-ld#context [HTTP Link Header](https://tools.ietf.org/html/rfc8288#section-3) MUST be ignored for such documents.

So far, this statement applied to VC documents in JSON-LD, so we were fine. However, the official media type is now `/vc`, meaning that the JSON-LD statement does not apply anymore.

(Actually, a possibility might be to take over the JSON-LD statement and add it to the media type specification...)

Reply:
There are two problems here:

1. That guidance in JSON-LD 1.1 (rel-linking to a context) seems to be something that is rarely, if ever used. It seems to be creating unnecessary complexity without much gain.
2. We don't use the rel-linking style anywhere in VCDM and DI, and should probably keep it that way (to avoid unnecessary complexity).

All that to say, we should probably fix JSON-LD v1.1 and not further complicate VCDM or DI with that rel-linking feature.

Reply:
> There are two problems here:
> 
> 1. That guidance in JSON-LD 1.1 (rel-linking to a context) seems to be something that is rarely, if ever used. It seems to be creating unnecessary complexity without much gain.

Maybe. But "hardly ever used" is a bit vague for the JSON-LD WG to act upon. At this point, if the JSON-LD WG proposed to remove the feature, at the minimum it should ask the community to see if there are users of the feature, ie, whether removing it would break applications. In the absence of this, I would be opposed to remove it.

> 2. We don't use the rel-linking style anywhere in VCDM and DI, and should probably keep it that way (to avoid unnecessary complexity).

I actually agree. But while that was automatically the case so far, it is not anymore 😒.

> 
> All that to say, we should probably fix JSON-LD v1.1 and not further complicate VCDM or DI with that rel-linking feature.

This is where we disagree. We have a problem _now_, and it is very uncertain that the JSON-LD WG will change that in the near future. I believe we _must_ document this restriction.

> I'm presuming your comment is a no-op on this PR, @iherman... if you think we should do something about it, it probably deserves an issue for discussion.

Yes, I agree it is a no-op on this PR. If we agree that the best place to handle this is to add a remark to the media type documentations (i.e., [Appendix C of the VCDM spec](https://www.w3.org/TR/vc-data-model-2.0/#iana-considerations)) then we should move ahead with a PR on the VCDM right away. If you prefer to do it, fine, if you want me to do it, I can try.

Reply:
Actually, @msporny, there is a statement in the media type specification:

> A [[JSON-LD11](https://www.w3.org/TR/vc-data-model-2.0/#bib-json-ld11)] context is expected to be present in the body of the document. The credential is expected to be a valid [JSON-LD document](https://www.w3.org/TR/json-ld11/#dfn-json-ld-document).

Which may be interpreted as saying that the link is not allowed. I still believe making this explicit should be done.

Reply:
Sure, making it explicit would be helpful. Do you want to raise an issue/PR to do that, @iherman? or should I?

Reply:
> Sure, making it explicit would be helpful. Do you want to raise an issue/PR to do that, @iherman? or should I?

I will do it, probably tomorrow

Reply:
See https://github.com/w3c/vc-data-model/pull/1539, @msporny